IDEOorg/steps
# #161240500 Fix Failing Auth Tests

#### What does this PR do?
Fix failing authentication tests
#### Description of Task to be completed?
- Fix the failing Authenticate test in `admin/src/routes/Auth/Authenticate.test.tsx` 
- Fix the failing test in `admin/src/services/auth0.test.ts`
#### How should this be manually tested?
- Clone the repository from `https://github.com/IDEOorg/steps.git`
- Check out the branch `git checkout bg-161240500-fix-failing-auth-tests`
- Follow the instruction here `https://github.com/IDEOorg/steps#refresh-with-latest-build` to install the dependencies
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#161240500](https://www.pivotaltracker.com/story/show/161240500)
#### Screenshots (if appropriate)
#### Questions:
